src/GslCore/GslCore.fsproj

```diff
@@ -9,12 +8,8 @@
   </PropertyGroup>
   <Import Project="..\..\packages\FsLexYacc\build\FsLexYacc.targets" />
   <ItemGroup>
-    <FsYacc Include=".\GslParser.fsy">
-      <OtherFlags>--module GslParser -o GslParser.fs</OtherFlags>
-    </FsYacc>
-    <FsLex Include=".\GslLexer.fsl">
-      <OtherFlags>--unicode -o GslLexer.fs</OtherFlags>
-    </FsLex>
+    <FsLex Include="GslLexer.fsl" />
+    <FsYacc Include="GslParser.fsy" />
     <Compile Include="Constants.fs" />
```

Comment:
Just checking on this change - this removed the params to the lexer / yacc which is perhaps unintended?